var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -52,16 +52,12 @@ class Openblas(Package):
     patch('make.patch')
 
     @property
-    def blas_libs(self):
+    def libs(self):
         shared = True if '+shared' in self.spec else False
         return find_libraries(
             ['libopenblas'], root=self.prefix, shared=shared, recurse=True
         )
 
-    @property
-    def lapack_libs(self):
```

Comment:
@davydden  Limited in which sense ? Suppose `mkl` is used, then if you code : 
```python
l = spec['scalapack'].libs
```
this will in order try to grab / call:
```
Mkl.scalapack_libs
Mkl.libs
_libs_default_handler
```
If you need specialization you can code : 
```python
class Mkl(Package):
   @property
   def scalapack_libs(self):
      ...

   @property
   def lapack_libs(self):
      ...


```
and take appropriate measures based on what you were asked.

There's a little caveat in that I didn't take care so far to return copied specs on `__getitem__` call, so that things like : 
```python
a = spec['blas']
b = spec['lapack']
...
```
don't interfere with each other if they are provided by the same package. That is a detail I'll work on later if this syntax / machinery looks good enough...